learn/advanced/updating.md

```diff
@@ -201,59 +147,222 @@ The server should return a response that looks like this:
+curl "https://github.com/meilisearch/meilisearch/releases/download/{meilisearch_version}/meilisearch-linux-amd64" --output meilisearch --location --show-error
+```
+
+:::
+
+::::
+
+Give execute permission to the Meilisearch binary:
+
+```
+chmod +x meilisearch
+```
+
+For **cloud platforms**, move the new Meilisearch binary to the `/usr/bin` directory containing the executable files:
+
+```
+mv meilisearch /usr/bin/meilisearch
 ```
 
 ::: warning
 If you are using Meilisearch v0.20 or below, migration should be done in two steps. First, import your dump into an instance running any version of Meilisearch from v0.21 to v0.24, inclusive. Second, export another dump from this instance and import it to a final instance running your targeted version.
+
+Once Meilisearch v1 is released, this two-step process won't be necessary as v1 will be compatible with dumps from all previous versions.
 :::
 
+### Launch Meilisearch and import the dump
+
+Execute the command below to import the dump at launch:
+
+:::: tabs
+
+::: tab Local installation
+
+```bash
+# replace {dump_uid.dump} with the name of your dump file
+./meilisearch --import-dump dumps/{dump_uid.dump} --master-key="MASTER_KEY"
+```
+
+:::
+
+::: tab Cloud platforms
+
+```sh
+# replace {dump_uid.dump} with the name of your dump file
+meilisearch --db-path /var/lib/meilisearch/data.ms --import-dump "/var/opt/meilisearch/dumps/{dump_uid.dump}"
+```
+
+:::
+
+::::
+
 Importing a dump requires indexing all the documents it contains. Depending on the size of your dataset, this process can take a long time and cause a spike in memory usage.
 
-Finally, don’t forget to set `displayedAttributes` back to its previous value if necessary. You can do this using the [update displayed attributes endpoint](/reference/api/settings.md#update-displayed-attributes).
+### Restart Meilisearch as a service
 
-Congratulations! You have successfully migrated your Meilisearch database to the latest version! 🎉
+If you're running a **cloud instance**, press `Ctrl`+`C`  to stop Meilisearch once your dump has been correctly imported. Next, execute the following command to run the script to configure Meilisearch and restart it as a service:
+
+```
+meilisearch-setup
+```
+
+If required, set `displayedAttributes` back to its previous value using the [update displayed attributes endpoint](/reference/api/settings.md#update-displayed-attributes).
+
+### Delete backup files or rollback (_optional_)
+
+Now that your updated Meilisearch instance is up and running, verify that the dump import was successful and no data was lost.
+
+If for some reason, something went wrong, you can always roll back to the previous version. Move the files back to their previous location using:
+
+:::: tabs
+
+::: tab Local installation
+
+```
+mv /tmp/meilisearch /path/to/your/meilisearch/directory/meilisearch
+mv /tmp/data.ms /path/to/your/meilisearch/directory/meilisearch/data.ms
```

Comment:
May need to update this command depending on how the previous command is changed (i.e. whether or not `data.ms` stays within the `meilisearch` directory).